sdk-actors/pom.xml

```diff
@@ -67,6 +67,11 @@
       <version>${grpc.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-validator</groupId>
```

Comment:
We cannot add 3rd party dependencies. These can conflict with app dependencies. If this is needed for tests only, update the scope.